redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -334,5 +336,310 @@ inline void processHardwareIsolationReq(
+                    {
+                        const AssociationsValType* associations =
+                            std::get_if<AssociationsValType>(&property.second);
+                        if (associations == nullptr)
+                        {
+                            BMCWEB_LOG_ERROR(
+                                "Failed to get the Associations from object: {}",
+                                hwStatusEventObj);
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+
+                        for (const auto& assoc : *associations)
+                        {
+                            if (std::get<0>(assoc) == "error_log")
+                            {
+                                sdbusplus::message::object_path errPath =
+                                    std::get<2>(assoc);
+                                // we have only one condition
+                                nlohmann::json_pointer<nlohmann::json>
+                                    logEntryPropPath(
+                                        "/Status/Conditions/0/LogEntry");
+                                error_log_utils::setErrorLogUri(
+                                    asyncResp, errPath, logEntryPropPath, true);
+                            }
+                        }
+                    }
+                    else if (property.first == "Timestamp")
+                    {
+                        const uint64_t* timestamp =
+                            std::get_if<uint64_t>(&property.second);
+                        if (timestamp == nullptr)
+                        {
+                            BMCWEB_LOG_ERROR(
+                                "Failed to get the Timestamp from object: {}",
+                                hwStatusEventObj);
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                        condition["Timestamp"] =
+                            redfish::time_utils::getDateTimeStdtime(
+                                static_cast<std::time_t>(*timestamp));
+                    }
+                    else if (property.first == "Message")
+                    {
+                        const std::string* msgPropVal =
+                            std::get_if<std::string>(&property.second);
+                        if (msgPropVal == nullptr)
+                        {
+                            BMCWEB_LOG_ERROR(
+                                "Failed to get the Message from object: {}",
+                                hwStatusEventObj);
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+
+                        const redfish::registries::Message* msgReg =
+                            registries::getMessage(
+                                "OpenBMC.0.2.HardwareIsolationReason");
```

Comment:
Shouldn't this match the message registry version that added this message? (i.e. OpenBMC.0.5.0)

Reply:
Will address this as part of JIRA